app/scripts/migrations/037.js

```diff
@@ -0,0 +1,46 @@
+// next version number
+const version = 36
+const { getFinalStates, containsFinalStates } = require('../controllers/transactions/lib/util')
+
+/*
+
+
+*/
+
+const clone = require('clone')
+
+module.exports = {
+  version,
+
+  migrate: async function (originalVersionedData) {
+    const versionedData = clone(originalVersionedData)
+    versionedData.meta.version = version
+    const state = versionedData.data
+    const newState = transformState(state)
+    versionedData.data = newState
+    return versionedData
+  },
+}
+
+function transformState (state) {
+  const newState = state
+  // transform state here
+  if (newState.TransactionController) {
+    if (newState.TransactionController.transactions) {
+      const transactions = newState.TransactionController.transactions
+      let txCount = transactions.length
+      if (txCount > 40) {
```

Comment:
Can we move this and the 40 below into some constant? e.g. `MAX_TX_HISTORY_LENGTH`?